### ISSUE TITLE
Move written docs to radareorg/cutter.re

### ISSUE THREAD

**Issue by ITAYC0HEN** [radareorg/cutter#2056]
**Is your feature request related to a problem? Please describe.**

Currently, the written documentation (features, shortcuts, how to build a plugin,...) is laying inside radareorg/cutter. While currently the documentation is short, we are mature and stable enough to make the first steps toward a more comprehensive documentation, and this is currently work in progress. 
The upcoming documentation will likely contain more pages and text, in addition to dozens of images and screenshots. There is no need that all this data will be on Cutter repository. 

**Describe the solution you'd like**

Move all the documentation, except the API docs, to radareorg/cutter.re

**Comment by thestr4ng3r** [comment]
What if we make a 3rd repo "cutter-docs" that has all the written docs and includes the regular cutter repo as a submodule.
Every master push to it re-generates the html docs and pushes it to cutter.re.
And every push to master on cutter updates the submodule on cutter-docs.

**Comment by ITAYC0HEN** [comment]
Can't it work with rdareorg/cutter.re without a 3rd one?

**Comment by thestr4ng3r** [comment]
It might work too if the sphinx generation process is moved from cutter's ci to cutter.re.

**Comment by thestr4ng3r** [comment]
However I think it should be possible to generate the documentation locally without cloning the website repo since by itself it has nothing to do with it.

**Comment by karliss** [comment]
I agree. Documentation and website are two different things. We might decide to redo the website from scratch that shouldn't affect documentation. We might also want to include documentation as part of binary packages in which case it would be generated as part of build process.
